views/frontend/post.php

```diff
@@ -32,7 +32,9 @@
                             </blockquote>
                         <?php
                     }
-                }else echo "Aucun commentaire n'a été publié... Soyez le premier!";
+                }else{ 
+                    echo "Aucun commentaire n'a été publié... Soyez le premier!";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5347246)